src/crypto/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# #
```

Comment:
It should be part of common lib. Mbedtls should be linked always for tests.

Reply:
Done